index.html

```diff
@@ -1923,10 +1924,47 @@ <h2>Arabic</h2>
 <div id="out_urdu-abjad" class="browser_output"></div>
 <script>addExamples('urdu-abjad', 'ur', 'rtl')</script>
 </div>
--->
 </section>
 
 
+<p class="note">We have reports of an alternative abjad style for Urdu that replaces  <span lang="ur" dir="rtl">ک</span>, <span lang="ur" dir="rtl">ل</span>, and <span lang="ur" dir="rtl">م</span> for 11, 12, 13 with <span lang="ur" dir="rtl">یا</span>, <span lang="ur" dir="rtl">یب</span>, and <span lang="ur" dir="rtl">یج</span>, respectively. These two styles may be used in the same publication, such as one in the front matter, the other in the preface.</p>
+
+
+
+
+
+
+
+
+
+
+
+
+<div class="csWrapper">
+<div class="copyMe" onclick="copyTemplate('urdu-alphabetic')"><img src="copy.svg" alt="Copy the style template." title="Copy the style template."></div>
+
+<p class="name" id="urdu-alphabetic"><a href="#urdu-alphabetic">urdu-alphabetic</a></p>
+
+<p class="counterstyle"><code><bdo dir="ltr">
+@counter-style urdu-alphabetic {
+system: fixed;
+symbols: '\0627' '\0628' '\067E' '\062A' '\0679' '\062B' '\062C' '\0686' '\062D' '\062E' '\062F' '\0688' '\0630' '\0631' '\0691' '\0632' '\0698' '\0633' '\0634' '\0635' '\0636' '\0637' '\0638' '\0639' '\063A' '\0641' '\0642' '\06A9' '\06AF' '\0644' '\0645' '\0646' '\06BA' '\0648' '\06C1' '\06BE' '\0621' '\06CC';
+/* symbols: '&#x0627;' '&#x0628;' '&#x067E;' '&#x062A;' '&#x0679;' '&#x062B;' '&#x062C;' '&#x0686;' '&#x062D;' '&#x062E;' '&#x062F;' '&#x0688;' '&#x0630;' '&#x0631;' '&#x0691;' '&#x0632;' '&#x0698;' '&#x0633;' '&#x0634;' '&#x0635;' '&#x0636;' '&#x0637;' '&#x0638;' '&#x0639;' '&#x063A;' '&#x0641;' '&#x0642;' '&#x06A9;' '&#x06AF;' '&#x0644;' '&#x0645;' '&#x0646;' '&#x06BA;' '&#x0648;' '&#x06C1;' '&#x06BE;' '&#x0621;' '&#x06CC;'; */
+}
+</bdo></code></p>
+
+<div id="out_urdu-alphabetic" class="browser_output"></div>
+<script>addExamples('urdu-alphabetic', 'ur', 'rtl')</script>
+</div>
+
+
+
+
+<aside class="note">
+<p>The urdu-alphabetic style is shown here as a fixed style.  According to CDAC, it is actually an unlimited style, where characters are doubled after the initial set is exhausted, eg. <span dir="rtl" lang="ur">ا ا</span>, <span dir="rtl" lang="ur">ب ب</span>, <span dir="rtl" lang="ur">ت ت</span>, etc. This can't be achieved using the standard 'alphabetic' algorithm. Furthermore, counters in this range do not join cursively, but are separated by a space. Until we have clearer evidence about how this works we list the style as fixed in length.</p>
```

Comment:
It would be useful to provide a full expansion of [C-DAC](https://www.cdac.in/).